### DISCUSSION TITLE
Where to look when RavenDb seems slow?

### DISCUSSION THREAD

**Discussion by ryanheath** [ravendb/ravendb#15444] [General]
We are using the ravendb cloud, and say 80-90% of the responses are with in milliseconds. Very great performance!

But ... occasionally we see responses of seconds with two digits, not so good.

I have looked into the admin logs in the studio but its downloads doesn't seem to have much information.
Where else can we look?

// Ryan

**Comment by ayende** [comment]
Start from getting a debug package of the system when this happens.

Is this happening on a particular period of time? Is there high load for some reason?
Or is this about one or two requests while others (in the same time frame) are fine?

Do you have any alerts or notifications in the studio?

If you can provide the timing for that, you can check the metrics in the cloud portal which may show more insight.
